src/table/paginator.scss

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../icon/index';
+@import '../button/index';
+@import './variables';
+
+@mixin iui-paginator {
+  @include iui-reset;
+  @include iui-font-family;
+  display: flex;
+  flex: 1;
+  justify-content: space-around;
+  align-items: center;
+  min-height: $iui-baseline * 5;
+  padding: 0 $iui-m;
+  user-select: none;
+  @include themed {
+    background-color: t(iui-color-background-3);
+  }
+
+  > .iui-left,
```

Comment:
sure no way to have it without left? What I understood its an empty element...

Reply:
I created `.iui-left` similar to `.iui-center` in the header.  I don't have anything to put in there currently, but the space is available incase someone needs it I suppose.